packages/external-db-config/lib/readers/aws_auth_config_reader.js

```diff
@@ -0,0 +1,36 @@
+const { SecretsManagerClient, GetSecretValueCommand } = require('@aws-sdk/client-secrets-manager')
+const { checkRequiredKeys } = require('../utils/config_utils')
+
+const EmptyAWSAuthConfig = { callbackurl: '', clientid: '', clientsecret: '', clientdomain: '' }
+
+class AwsAuthConfigReader {
```

Comment:
do you want to define a different secret key for auth ? 
i want to use the same key as before but auth will be a sub-property in the current config.

Reply:
I can't set the value for `clientsecret`, it's a secret that generated when I create the login screen in AWS/GCP

Reply:
![image](https://user-images.githubusercontent.com/407079/150677703-ae1d1426-496c-4fac-90fe-85772036ba28.png)